.github/workflows/security-testing.yml

```
@@ -14,125 +14,37 @@ env:
-
-    - name: Run audit log tests
-      id: auditlog
+        sudo apt-get update
+        sudo apt-get install -y \
+          cmake \
+          qt6-base-dev \
+          libpolkit-qt6-1-dev \
+          pkg-config
+
+    - name: Build and run unit tests
       run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2046:warning:4:8: Quote this to prevent word splitting

(shellcheck)